src/test/java/entities/AverageRatingsTest.java

```diff
@@ -7,23 +7,25 @@
 
 class AverageRatingsTest {
 
+    static AverageRatings r = new AverageRatings();
```

Comment:
maybe rename this to something that gives some more information? ex. averageRatings